pkg/square/builder.go

```diff
@@ -344,7 +344,7 @@ func (b *Builder) NumPFBs() int {
 }
 
 func (b *Builder) NumTxs() int {
-	return len(b.txs)
+	return len(b.txs) + len(b.pfbs)
 }
```

Comment:
[optional] this function may warrant a clarifying godoc

```go

// NumTxs returns the total number of txs (PFBs + normalTxs)
func (b *Builder) NumTxs() int {
	return len(b.txs) + len(b.pfbs)
}
```

we may also consider renaming the field `b.txs` => `b.normalTxs`

Reply:
I'm not 100% convinced about normal txs. I don't like that normality is defined on the basis of whether there is a PFB or not

Reply:
I'm open to other names to help disambiguate. Candidates: `nonPFBTxs`